src/lang/Messages.properties

```diff
@@ -2032,6 +2032,7 @@ view.options.label.responsepanelpos.above   = Request Shown Above Response
 view.options.label.responsepanelpos.sideBySide = Panels Side by Side
 view.options.label.responsepanelpos.tabs    = Tabs Side by Side
 view.options.label.showSplashScreen			= Show splash screen
+view.options.label.showlocalconnectrequests = Persist and show (local) CONNECT requests
```

Comment:
Yeah, will change that (I'll explain the behaviour more thoroughly in the user guide).